code/Code/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!--this links css to html-->
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+
+  <body>
+    <!--NAVIGATION-->
+    <menu class="menu-parent">
+      <img src="./Images/Yogadia_Logo.jpg" alt="Logo" class="menu-logo" />
+
+      <div class="menu-child 2">
+        <a href="./Klassen.html">KLASSEN</a>
+        <a href="./Abos.html">ABOS</a>
+        <a href="./Über_mich.html">ÜBER MICH</a>
+        <a href="./Kontakt.html">KONTAKT</a>
+      </div>
+    </menu>
+
+    <!--HERO-->
+    <header>
+      <div class="hero-picture">
+        <img
+          src="Images/Hero1_v1.jpg"
+          alt="Hero"
+          style="width: 100%; height: 125%"
+        />
+        <div class="hero-text">
+          <h1 class="h1">STARTE JETZT DEINE YOGA JOURNEY</h1>
+          <h2 class="h2">Lorem ipsum dolor sit amet, consectetur adipisici</h2>
+          <button class="button1">online buchen</button>
+        </div>
+      </div>
+    </header>
+
+    <!--BENEFITS-->
+
+    <!--ANGEBOT-->
+    <div class="angebot-titel">
+      <h1>YOGA ANGEBOT</h1>
+    </div>
+
+    <section class="section1">
+      <div class="spalte 1">
+        <h2>INDOOR VINYASA YOGA</h2>
+        <img src="images/Angebot1.jpg" width="350px" height="320px" />
+        <p>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy
+        </p>
+        <button class="button2">jetzt buchen</button>
+      </div>
+
+      <div class="spalte 2">
+        <h2>OUTDOOR YOGA</h2>
+        <img src="images/Angebot4.jpg" width="350px" height="320px" />
+        <p>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy
+        </p>
+        <button class="button2">jetzt buchen</button>
+      </div>
+
+      <div class="spalte 3">
+        <h2>YOGA FÜR FIRMEN</h2>
+        <img src="images/Angebot3.jpg" width="350px" height="320px" />
+        <p>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy
+        </p>
+        <button class="button2">jetzt buchen</button>
+      </div>
+    </section>
+
+    <!--ÜBER MICH-->
+    <section></section>
+    <section></section>
```

Comment:
I assume you want to add something here eventually, but in the meantime, it would be better to not leave empty tags.